indexer/src/main/java/org/uniprot/store/indexer/crossref/readers/CrossRefUniProtCountReader.java

```diff
@@ -15,20 +15,25 @@
 public class CrossRefUniProtCountReader
         implements RowMapper<CrossRefUniProtCountReader.CrossRefProteinCount> {
 
+    public static final String QUERY_TO_GET_XREF_PROTEIN_COUNT =
```

Comment:
Can you please move this constant to CrossRefUniProtCountStep that is the class that is using it.. This way you can change it to private.